src/main/scala/rocketchip/Periphery.scala

```diff
@@ -349,8 +349,7 @@ trait HasPeripheryErrorSlave extends HasSystemNetworks {
 
   // Override the default Parameters to exclude the TLMonitor between the Fragmenter and error slave.
```

Comment:
This comment could be updated now that its not an "override" of parameters.

"Most slaves...than others, therefore we exclude the TLMonitor."